src/components/crop-content/CropContent.tsx

```diff
@@ -34,7 +36,15 @@ export default class CropContent extends React.Component<CropContentProps> {
   }
 
   onToggle() {
-    this.setState({ collapsed: !this.state.collapsed });
+    const { onToggle } = this.props;
+    const { collapsed } = this.state;
+    
+    const reverseCollapsed = !collapsed;
+    this.setState({ collapsed: reverseCollapsed });
+
+    if (onToggle) {
+      onToggle(this.containerElRef, reverseCollapsed);
```

Comment:
I'd put the element second. 
In case somebody needs just the toggleStatus he could pass a function onToggle(isToggled)